tests/test_database_manager.py

```diff
@@ -19,6 +19,9 @@ def setUpClass(self):
 
     def setUp(self):
         setup_test.create_test_database_setup(self.database_connector)
+    
+    def tearDown(self):
+        setup_test.tear_down_database_setup(self.database_connector)
 
```

Comment:
This indentation looks weird? Or is it just me?

Reply:
looks fine for me